torch/testing/_internal/opinfo/definitions/_masked.py

```diff
@@ -456,11 +456,6 @@ def sample_inputs_masked_normalize(op_info, device, dtype, requires_grad, **kwar
                 "TestCudaFuserOpInfo",
                 "test_nvfuser_correctness",
             ),
-            DecorateInfo(
```

Comment:
Thanks for double checking these.

Reply:
I'll remember to diff these against upstream and we'll enable these failing tests when we bump things.